lib/ui/BUILD.gn

```diff
@@ -187,6 +187,7 @@ if (enable_unittests) {
     dart_main = "fixtures/ui_test.dart"
     fixtures = [
       "fixtures/DashInNooglerHat.jpg",
+      "fixtures/DashInNooglerHat%20WithSpace.jpg",
```

Comment:
Since the fixtures are bundled as-is using file path as key, we need to simulate flutter tool encoding strategy by changing the file name.